x-pack/platform/plugins/shared/onechat/public/application/context/send_message_context.tsx

```diff
@@ -131,7 +131,7 @@ const useSendMessageMutation = ({ connectorId }: UseSendMessageMutationProps = {
       });
     },
     onSettled: () => {
-      conversationActions.invalidateConversation();
+      conversationActions.removeNewConversationQuery();
```

Comment:
Maybe we still want to have the `invalidateConversation()` method here? Am I missing something? Why do we want to _invalidate_ the cache here?